src/main/java/uk/gov/ons/ctp/response/sample/party/PartyUtil.java

```diff
@@ -32,8 +32,12 @@ public static PartyCreationRequestDTO convertToParty(SampleUnitBase unit) throws
       businessSampleUnit.setRusic92(bsu.getRusic92());
       businessSampleUnit.setFrosic2007(bsu.getFrosic2007());
       businessSampleUnit.setRusic2007(bsu.getRusic2007());
-      businessSampleUnit.setFroempment(Integer.valueOf(bsu.getFroempment()));
-      businessSampleUnit.setFrotover(Integer.valueOf(bsu.getFrotover()));
+      if (!bsu.getFroempment().equals("")) {
```

Comment:
Not sure it s correct ... if you have a String " " what would you do ? You should hav eused isEmpty() or StringUtils.isEmpty()

Reply:
It might even be better to use http://commons.apache.org/proper/commons-lang/javadocs/api-release/org/apache/commons/lang3/StringUtils.html#isNumeric-java.lang.CharSequence-

Reply:
I'll implement Ben's recommendation one, looks ideal.
If we are given something non numeric we let the value default to 0 which is what we want here